input/pagecontent/reference.md

```diff
@@ -2730,6 +2734,42 @@ Any FSH syntax errors that arise as a result of the value substitution are handl
   // more rules
   ```
 
+{%include tu-div.html%}
+* Use the parameterized rule set, Variable, to add variable definitions to an instance of TestScript:
```

Comment:
I said this morning that I'd look at the examples to determine if they seem realistic and/or if there are better examples (such as post-coordinated SNOMED codes). After doing that, my determination is that these examples are good ones -- and better than anything else I could find!

Reply:
Users are not familiar with the TestScript resource. Ideally, examples should use common Resources. But I understand you couldn't come up with one. OK, but  RulesSet named "Variable" was confusing to me. Can you rename this to something more self-explanatory? I suggest `AddVariableToTestScript`.

Reply:
It looks like this is still not implemented.